lib/warren/message/simple.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Warren
+  # Namespace for Warren message wrappers.
+  module Message
+    # A simple message simply wraps the routing key and payload together
+    Simple = Struct.new(:routing_key, :payload, :headers)
+  end
+end
```

Comment:
Could only see this currently being used in the tests - maybe it could be moved out of lib?

Reply:
Its used in the delay exchange https://github.com/sanger/warren/blob/3b657809f5d8aa3ec06649a7c0bbf131ae638a07/lib/warren/delay_exchange.rb#L49-L49, but also its quite useful as a utility class for anything using the gem. Otherwise there is no 'lightweight' message provided, and the only option is to roll your own.